Tests/MaterialLib/TestFluidViscosity.cpp

```diff
@@ -124,3 +126,48 @@ TEST(Material, checkVogelViscosity)
     ASSERT_NEAR(0.352072e-4, mu_ch4->getValue(vars), 1.e-5);
     ASSERT_NEAR(-2.35664e-6, mu_ch4->getdValue(vars, var_type), 1.e-5);
 }
+
+TEST(Material, checkWaterViscosityIAPWS)
+{
+    const char xml_w[] =
+        "<viscosity>"
+        "  <type>WaterViscosityIAPWS</type>"
+        "</viscosity>";
+    const auto mu_w = createTestViscosityModel(xml_w);
+    const double T[] = {298.15, 298.15, 373.15,  433.15,  433.15, 873.15,
+                        873.15, 873.15, 1173.15, 1173.15, 1173.15};
+    const double rho[] = {998, 1200, 1000, 1, 1000, 1, 100, 600, 1, 100, 400};
+
+    const double expected_mumu[] = {
+        889.735100, 1437.649467, 307.883622, 14.538324, 217.685358, 32.619287,
+        35.802262,  77.430195,   44.217245,  47.640433, 64.154608};
+
+    const double purterbation = 1.e-9;
```

Comment:
It is used to calculate the numerical differentiation of mu with respect to  rho or T. Compared to the scale of rho and T, 1e-9 is small enough.